src/common/snippets/include/snippets/op/brgemm.hpp

```diff
@@ -38,6 +39,15 @@ class Brgemm : public MemoryAccess {
 
     bool has_evaluate() const override { return false; }
 
+    class ShapeInfer : public IShapeInferSnippets {
+        std::vector<std::vector<size_t>> m_io_layouts;
+        VectorDims get_reordered_planar_shape(const VectorDims & shape, const std::vector<size_t>& layout);
```

Comment:
We don't really need this method here, since we have `snippets::utils::lowered::get_planar_shape`.
I rewrote this ShapeInfer a little bit, please take a look